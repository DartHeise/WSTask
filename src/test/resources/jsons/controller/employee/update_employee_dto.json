{
  "firstName": "Artem",
  "lastName": "Kornev",
  "description": "consectetur adipiscing elit",
  "characteristics": [
    "shy",
    "tactful",
    "resourceful",
    "reliable"
  ],
  "postId": "854ef89d-6c27-4635-926d-894d76a81707",
  "jobType": "PERMANENT",
  "contacts": {
    "phone": "+16463483212",
    "email": "Kornevenrok@gmail.com",
    "workEmail": "KornevWorker123@bk.ru"
  }
}