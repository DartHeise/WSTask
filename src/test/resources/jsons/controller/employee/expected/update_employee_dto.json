{
  "id": "8ee05ef8-eed9-11ec-8ea0-0242ac120002",
  "firstName": "Artem",
  "lastName": "Kornev",
  "description": "consectetur adipiscing elit",
  "characteristics": [
    "shy",
    "tactful",
    "resourceful",
    "reliable"
  ],
  "postId": "854ef89d-6c27-4635-926d-894d76a81707",
  "jobType": "PERMANENT",
  "contacts": {
    "phone": "+16463483212",
    "email": "Kornevenrok@gmail.com",
    "workEmail": "KornevWorker123@bk.ru"
  }
}